packages/ui/src/components/RequestPanel.vue

```diff
@@ -351,6 +351,7 @@
             @click="handleRequestBodyMenuClick"
         />
     </template>
+    <HttpMethodModal v-model:showModal="httpMethodModalShow" @customHttpMethod="handleCustomHttpMethod"></HttpMethodModal>
```

Comment:
That's actually the emitted event listener from the component. You'll notice that it will work even if you remove the property from data. They're not connected in any way.